pkg/rbd/rbd_util.go

```diff
@@ -109,36 +116,98 @@ type rbdSnapshot struct {
 
 var (
 	supportedFeatures = sets.NewString("layering")
+
+	// large interval and timeout, it should be longer than the maximum
+	// time an operation can take (until refcounting of the connections is
+	// available)
+	connPool = util.NewConnPool(15*time.Minute, 10*time.Minute)
 )
 
 // createImage creates a new ceph image with provision and volume options.
 func createImage(ctx context.Context, pOpts *rbdVolume, volSz int64, cr *util.Credentials) error {
-	var output []byte
-
-	image := pOpts.RbdImageName
 	volSzMiB := fmt.Sprintf("%dM", volSz)
+	options := librbd.NewRbdImageOptions()
+
+	imageFormat, err := strconv.Atoi(rbdImageFormat2)
```

Comment:
can we avoid this conversion and define `rbdImageFormat2` as int?

Reply:
Yeah, I'll probably do this. Will need to see if it is used in other places.